aws-audit.py

```diff
@@ -71,6 +73,14 @@ def parse_billing_data(billing_data):
       user_dict[acct_num]['total'] = float(row[24])
       user_dict[acct_num]['currency'] = row[23]
 
+  # now apply any credits received (for things like compromised accounts)
+  for row in billing_data:
+    if len(row) < 4:
+      continue
+    if row[3] == 'LinkedLineItem' and 'Unauthorized Usage' in row[18]:
```

Comment:
> `'Unauthorized Usage' in row[18]`

might want to use a stricter test than just a substring check, since it's better for the script to error out if the format of the field changes, which serves as a notice to update the code for the new changes